src/components/PrimaryNav/PrimaryNav.scss

```diff
@@ -3,7 +3,7 @@
 $primaryNavExpanded: 300px;
 $primaryNav: 56px;
 
-.logoCaptionGroup {
+.moonstone-logoCaptionGroup {
```

Comment:
It's not directly related to this PR, but the class naming here is invalid, they are not prefix with the block name . All classes should start with moonstone-primaryNav_*

Reply:
Ok! I've updated a bunch of class names on this component.